src/main/java/net/sacredlabyrinth/phaed/simpleclans/ui/frames/staff/PlayerListFrame.java

```diff
@@ -63,10 +63,8 @@ private void loadPlayers() {
 		if (onlineOnly) {
 			players = new ArrayList<>(Bukkit.getOnlinePlayers());
 		} else {
-			players = clanManager.getAllClanPlayers().stream().map(ClanPlayer::getUniqueId)
-					.map(Bukkit::getOfflinePlayer).collect(Collectors.toList());
-			List<Player> onlinePlayers = Bukkit.getOnlinePlayers().stream().filter(p -> !players.contains(p))
-					.collect(Collectors.toList());
+			List<Player> onlinePlayers = clanManager.getOnlineClanPlayers().stream().
```

Comment:
This part should list all players (online and offline). (Only online is above)

Reply:
players = getOfflineClanMembers
onlinePlayers = getOnlinePlayers().filter -> if OfflineClanMembers doesn't contains OnlinePlayer -> returns Players, which are not contains contains in OfflineClanMembers (so, which are ONLINE).